Files/ViewModels/PreviewPaneViewModel.cs

```diff
@@ -98,11 +98,7 @@ private async Task LoadPreviewControlAsync()
 
             if (SelectedItem.PrimaryItemAttribute == StorageItemTypes.Folder)
             {
-                // TODO: Finish folder previews and reimplement later
-                //PreviewPaneContent = new FolderPreview(new FolderPreviewViewModel(SelectedItem));
-                DetailsErrorText = "PreviewPaneDetailsNotAvailableText".GetLocalized();
-                PreviewErrorText = "DetailsPanePreviewNotAvaliableText".GetLocalized();
```

Comment:
Can these strings be removed?

Reply:
@yaichenbaum No, they're used elsewhere

Those are the "No preview available" and "No details available" strings.